HEN_HOUSE/src/egsnrc.mortran

```diff
@@ -1453,6 +1466,7 @@ $start_new_particle;
                 "default for $DE-FLUCTUATION; is ; (null)
             edep = de; "energy deposition variable for user
             $ADD-WORK-EM-FIELD;  "e-loss or gain in em field
```

Comment:
There are now two macros here for adding work; the proper one ought to be selected based on `$EM_MACROS_ACTIVE`?

Reply:
It is fine, because each macro gets defined upon inclusion of the corresponding EM macros, either `emf_macros.mortran` or `EEMF_macros.mortran`; both macro files would not be included in the compilation chain at the same time!